public/assets/.sprockets-manifest-96997a16c796b4f354d6a0146b970e7a.json

```diff
@@ -0,0 +1 @@
+{"files":{"application-a51a53658dfaa975cb4363f51d14bccf24d66c3d90c2186bf834c157151c4bd1.js":{"logical_path":"application.js","mtime":"2018-02-24T22:34:29+02:00","size":84893,"digest":"a51a53658dfaa975cb4363f51d14bccf24d66c3d90c2186bf834c157151c4bd1","integrity":"sha256-pRpTZY36qXXLQ2P1HRS8zyTWbD2Qwhhr+DTBVxUcS9E="},"application-f0d704deea029cf000697e2c0181ec173a1b474645466ed843eb5ee7bb215794.css":{"logical_path":"application.css","mtime":"2018-02-24T22:34:29+02:00","size":676,"digest":"f0d704deea029cf000697e2c0181ec173a1b474645466ed843eb5ee7bb215794","integrity":"sha256-8NcE3uoCnPAAaX4sAYHsFzobR0ZFRm7YQ+te57shV5Q="}},"assets":{"application.js":"application-a51a53658dfaa975cb4363f51d14bccf24d66c3d90c2186bf834c157151c4bd1.js","application.css":"application-f0d704deea029cf000697e2c0181ec173a1b474645466ed843eb5ee7bb215794.css"}}
```

Comment:
Compiled assets mustn't come into PR. Please run these commands in terminal and push again.
```bash
rake assets:clean
rake assets:clobber
```